### PR TITLE
Fix Python Lint errors

### PR DESCRIPTION
**Pull request by hughbe**
I introduced some of these, so I thought I'd fix them

### CODE REVIEW

**Comment by hughbe** [comment]
@swift-ci please python lint

**Comment by hughbe** [comment]
@swift-ci please python lint

**Comment by hughbe** [comment]
@swift-ci please python lint

**Comment by hughbe** [comment]
@swift-ci please python lint

**Comment by hughbe** [comment]
@swift-ci please python lint

**Comment by hughbe** [comment]
@swift-ci please smoke test

File: utils/update-checkout [hunk 1]
```diff
@@ -101,7 +101,8 @@ def update_single_repository(args):
                 shell.run(["git", "rebase", "FETCH_HEAD"], echo=True)
             elif detached_head:
                 print(repo_path,
-                      "\nDetached HEAD; probably checked out a tag. No need to rebase.\n")
+                      "\nDetached HEAD; probably checked out a tag. No need to"
```
Comment (rintaro) [inline utils/update-checkout]: missing space before quote: `to "`

**Comment by hughbe** [comment]
@swift-ci please smoke test

**Comment by hughbe** [comment]
@swift-ci please python lint

**Comment by hughbe** [comment]
@swift-ci please smoke test

**Comment by hughbe** [comment]
@swift-ci please python lint

**Comment by hughbe** [comment]
@swift-ci please smoke test

**Comment by hughbe** [comment]
@swift-ci please python lint

**Comment by hughbe** [comment]
@swift-ci please python lint

**Comment by hughbe** [comment]
@swift-ci please smoke test

File: utils/update-checkout [hunk 2]
```diff
@@ -185,43 +187,49 @@ def update_all_repositories(args, config, scheme_name, cross_repos_pr):
...
-    args, config, with_ssh, scheme_name, skip_history, skip_repository_list):
+def obtain_all_additional_swift_sources(args, config, with_ssh, scheme_name,
+                                        skip_history, skip_repository_list):
 
     pool_args = []
     with shell.pushd(SWIFT_SOURCE_ROOT, dry_run=False, echo=False):
         for repo_name, repo_info in config['repos'].items():
             if repo_name in skip_repository_list:
-                print("Skipping clone of '" + repo_name + "', requested by user")
+                print("Skipping clone of '" + repo_name + "', requested by"
```
Comment (rintaro) [inline utils/update-checkout]: missing space: `by "`

File: utils/update-checkout [hunk 3]
```diff
@@ -185,43 +187,49 @@ def update_all_repositories(args, config, scheme_name, cross_repos_pr):
...
+                      "user")
                 continue
 
             if os.path.isdir(os.path.join(repo_name, ".git")):
-                print("Skipping clone of '" + repo_name + "', directory already exists")
+                print("Skipping clone of '" + repo_name + "', directory"
```
Comment (rintaro) [inline utils/update-checkout]: ditto

File: utils/update-checkout [hunk 4]
```diff
@@ -297,9 +310,9 @@ def validate_config(config):
     # This guarantees sensible behavior of update_repository_to_scheme when
     # the branch-scheme is passed as the scheme name
     for scheme_name in config['branch-schemes'].keys():
-        if not scheme_name in config['branch-schemes'][scheme_name]['aliases']:
-            raise RuntimeError('branch-scheme name: "{0}" must be an alias too.'
-                .format(scheme_name))
+        if scheme_name not in config['branch-schemes'][scheme_name]['aliases']:
+            raise RuntimeError('branch-scheme name: "{0}" must be an alias'
```
Comment (rintaro) [inline utils/update-checkout]: ditto

File: utils/update-checkout [hunk 5]
```diff
@@ -372,7 +386,8 @@ By default, updates your checkouts of Swift, SourceKit, LLDB, and SwiftPM.""")
         help='Dump the git hashes of all repositories being tracked')
     parser.add_argument(
         '--dump-hashes-config',
-        help='Dump the git hashes of all repositories packaged into update-checkout-config.json',
+        help='Dump the git hashes of all repositories packaged into'
```
Comment (rintaro) [inline utils/update-checkout]: ditto

File: utils/update-checkout [hunk 6]
```diff
@@ -387,7 +402,8 @@ By default, updates your checkouts of Swift, SourceKit, LLDB, and SwiftPM.""")
     args = parser.parse_args()
 
     if args.reset_to_remote and not args.scheme:
-        print("update-checkout usage error: --reset-to-remote must specify --scheme=foo")
+        print("update-checkout usage error: --reset-to-remote must specify"
```
Comment (rintaro) [inline utils/update-checkout]: ditto

Comment (hughbe) [inline utils/update-checkout]: Thanks, was too lazy to check more after your first comment, guess that's what happens!

**Comment by hughbe** [comment]
@swift-ci please python lint

**Comment by hughbe** [comment]
@swift-ci please python lint

**Comment by hughbe** [comment]
@swift-ci please smoke test

**Comment by gottesmm** [comment]
@hughbe A suggestion. The linter I think can run as a lint test in the validation tests. This will ensure that these never crop up again.

Look in ./validation-test/Python.

**Comment by hughbe** [comment]
@gottesmm good suggestion. Mind if I do that in a separate PR, this one will keep getting merge conflicts!
Just fixed a merge conflict, will wait for a signoff before merging

**Comment by hughbe** [comment]
@swift-ci please python lint

**Comment by hughbe** [comment]
@swift-ci please python lint

**Comment by hughbe** [comment]
@swift-ci please python lint

**Comment by hughbe** [comment]
@swift-ci please python lint

**Comment by hughbe** [comment]
@swift-ci please python lint

**Comment by hughbe** [comment]
@swift-ci please python lint

**Comment by hughbe** [comment]
@swift-ci please smoke test

**Comment by hughbe** [comment]
@swift-ci please smoke test

**Comment by gottesmm** [comment]
@hughbe SGTM!

**Comment by gottesmm** [comment]
@swift-ci Please smoke test linux platform

**Comment by gottesmm** [comment]
The test failure was an lldb failure. Not sure how this commit could have caused that... But lets try again.

**Comment by hughbe** [comment]
@practicalswift I have a quick question.

I'm also running `pylint` over the python source files - I've got some useful little cleanups so I'll submit a PR for that.

However, I get the suggestion to make global variables (i.e. constants), capital. Take the example of update-checkout [here](https://github.com/apple/swift/blob/59037e9fdc479e8b858b6f18cd33e5ad87697035/utils/update-checkout#L448)

```
if __name__ == "__main__":
    freeze_support()
    clone_results, update_results = main()
    fail_count = 0
    fail_count += shell.check_parallel_results(clone_results, "CLONE")
    fail_count += shell.check_parallel_results(update_results, "UPDATE")
    if fail_count > 0:
        print("update-checkout failed, fix errors and try again")
    sys.exit(fail_count)
```

I get the error
> C:445, 4: Invalid constant name "clone_results" (invalid-name)

Changing it `CLONE_RESULTS` works. However, do we want this behaviour that pep8 requires, or have we decided to willfully disobey the requirements for constants to be ALLCAPS?

**Pull request merged**